src/lib/Libifl/pbs_loadconf.c

```diff
@@ -278,7 +278,7 @@ frame_psi(psi_t *psi, char *svr_id)
 	if (svrname == NULL)
 		return -1;
 
-	strcpy(psi->name, svrname);
+	strncpy(psi->name, svrname, sizeof(psi->name));
```

Comment:
AFAIK, now a days we are using pbs_strncpy() instead strncpy() in new/modified code